src/locale/locales/en.json

```diff
@@ -86,5 +86,9 @@
   "SNTable.SelectionLabel.selected": {
     "value": "Selected",
     "comment": "announced by screen readers for a selected value. (tew 210929)"
+  },
+  "SNTable.Object.Table.Totals": {
```

Comment:
Remove for now. we might need a default string that is translated later, but it could be hard-coded for now